rivertest/worker_test.go

```diff
@@ -22,6 +22,40 @@ type testArgs struct {
 
 func (testArgs) Kind() string { return "rivertest_work_test" }
 
+func TestWorker_NewWorker(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+
+	type testBundle struct {
+		config *river.Config
+		driver *riverpgxv5.Driver
+		tx     pgx.Tx
+	}
+
+	setup := func(t *testing.T) *testBundle {
+		t.Helper()
+
+		return &testBundle{
+			config: &river.Config{ID: "rivertest-worker"},
+			driver: riverpgxv5.New(nil),
+			tx:     riverinternaltest.TestTx(ctx, t),
+		}
+	}
+
+	t.Run("HandlesANilRiverConfig", func(t *testing.T) {
```

Comment:
Nit, but found the casing a bit awkward to read. Really doesn't need the "A" anyway:

```suggestion
	t.Run("HandlesNilRiverConfig", func(t *testing.T) {
```